### DISCUSSION TITLE
any progress on this project?

### DISCUSSION THREAD

**Discussion by vanasoft23** [bangcorrupt/freetribe#1] [General]
Hi, I was just wondering if hackribe or freetribe is still being developed. If there was code available I'm quite interested in this project.
Electribe 2 has a lot of potential still

**Comment by bangcorrupt** [comment]
Hi, Hacktribe and Freetribe are still being developed, albeit quite slowly.  The next update to Hacktribe will be a loader, triggered via SysEx, allowing Freetribe apps to be loaded and executed via MIDI.  I have it working, it just needs tidying up a bit.  If I can, over the weekend I'll push the changes to Hacktribe and the Freetribe source to the public repo.  Dealing with a minor housing crisis so everything is in chaos.

**Reply by vanasoft23** [reply]
Ah okay life takes away lots of time as usual :)
I have some background in x86 and lot's of C but none in embedded systems. How did you get around and learn embedded systems so that you were able to reverse engineer the korg?
And do you work with a personally accumulated ghidra symbols/labels file, or did you somehow just know through code reading and debugging what things did?

I have quite a bunch of theoretically realistic improvements in my mind that can push the electribe workflow further
